addons/l10n_gr/data/template/account.tax-gr.csv

```diff
@@ -1,37 +1,331 @@
+"l10n_gr_tax_p6_other_imports","6% Import","Other imports apart from investment goods (capital goods) (VAT 6%)","","","6%","6.0","1","percent","purchase","l10n_gr_tax_group_6","False","base","invoice","","+363","","6% Εισαγωγή","Άλλες εισαγωγές εκτός από επενδυτικά αγαθά (κεφαλαιουχικά αγαθά) (ΦΠΑ 6%)"
+"","","","","","","","","","","","","tax","invoice","","+383","l10n_gr_54_02_02","",""
+"","","","","","","","","","","","","base","refund","","-363","","",""
+"","","","","","","","","","","","","tax","refund","","-383","l10n_gr_54_02_02","",""
+"l10n_gr_tax_p17_other_imports","17% Import","Other imports apart from investment goods (capital goods) (VAT 17%)","","","17%","17.0","1","percent","purchase","l10n_gr_tax_group_17","False","base","invoice","","+363","","17% Εισαγωγή","Άλλες εισαγωγές εκτός από επενδυτικά αγαθά (κεφαλαιουχικά αγαθά) (ΦΠΑ 17%)"
+"","","","","","","","","","","","","tax","invoice","","+383","l10n_gr_54_02_02","",""
+"","","","","","","","","","","","","base","refund","","-363","","",""
+"","","","","","","","","","","","","tax","refund","","-383","l10n_gr_54_02_02","",""
+"l10n_gr_tax_p9_other_imports","9% Import","Other imports apart from investment goods (capital goods) (VAT 9%)","","","9%","9.0","1","percent","purchase","l10n_gr_tax_group_9","False","base","invoice","","+363","","9% Εισαγωγή","Άλλες εισαγωγές εκτός από επενδυτικά αγαθά (κεφαλαιουχικά αγαθά) (ΦΠΑ 9%)"
+"","","","","","","","","","","","","tax","invoice","","+383","l10n_gr_54_02_02","",""
+"","","","","","","","","","","","","base","refund","","-363","","",""
+"","","","","","","","","","","","","tax","refund","","-383","l10n_gr_54_02_02","",""
+"l10n_gr_tax_p4_other_imports","4% Import","Other imports apart from investment goods (capital goods) (VAT 4%)","","","4%","4.0","1","percent","purchase","l10n_gr_tax_group_4","False","base","invoice","","+363","","4% Εισαγωγή","Άλλες εισαγωγές εκτός από επενδυτικά αγαθά (κεφαλαιουχικά αγαθά) (ΦΠΑ 4%)"
+"","","","","","","","","","","","","tax","invoice","","+383","l10n_gr_54_02_02","",""
+"","","","","","","","","","","","","base","refund","","-363","","",""
+"","","","","","","","","","","","","tax","refund","","-383","l10n_gr_54_02_02","",""
+"l10n_gr_tax_p0_other_imports","0% Import","Other imports apart from investment goods (capital goods) (VAT 0%)","","","0%","0.0","1","percent","purchase","l10n_gr_tax_group_0","False","base","invoice","","+363","","0% Εισαγωγή","Άλλες εισαγωγές εκτός από επενδυτικά αγαθά (κεφαλαιουχικά αγαθά) (ΦΠΑ 0%)"
+"","","","","","","","","","","","","tax","invoice","","","","",""
+"","","","","","","","","","","","","base","refund","","-363","","",""
+"","","","","","","","","","","","","tax","refund","","","","",""
+"l10n_gr_tax_p24_G_eu","24% EU G","Intra-community acquisitions of goods (VAT 24%)","consu","","24%","24.0","1","percent","purchase","l10n_gr_tax_group_24","","base","invoice","","+364","","","Ενδοκοινοτικές αποκτήσεις αγαθών (ΦΠΑ 24%)"
```

Comment:
You can specify multiple tax tags on the repartition lines like this: `"+364||+303"`